test/useForm.test.tsx

```diff
@@ -555,6 +557,88 @@ describe('useForm', () => {
     expect(isValid).toBeFalsy()
     expect(fields.arrayField.error).toEqual("This field can't be empty.")
   })
+
+  it('should not allow zero for positiveNumber rule by default', async () => {
+    type SpecificObject = { positiveNumber: number }
+
+    const { result } = render<SpecificObject>({
+      positiveNumber: positiveNumberField()
+    })
+
+    act(() => {
+      result.current.fields.positiveNumber.setValue(2)
+    })
+
```

Comment:
This is a small nitpick, but might be worth asserting there isn't an error at this point.